flake.nix

```
@@ -0,0 +1,22 @@
+{
+  description = "Development environment in a flake.nix file for Nix users";
+
+  inputs = {
+    nixpkgs.url = "nixpkgs/nixos-unstable";
+  };
+
+    outputs = { self, nixpkgs }:
+      let
+        system = "x86_64-linux";
+        pkgs = nixpkgs.legacyPackages.${system};
```

```suggestion
        pkgs = import nixpkgs { inherit system; };
```
idk if i have the best idea though but i think this works